WordPress/gutenberg
# [Block Library - Post Excerpt]: Add attribute to show the `read more` link

This is POC for adding a new attribute to `Post Excerpt` block to hide by default the `read more` link. This is because recently a new standalone[ Read More block](https://github.com/WordPress/gutenberg/pull/37649) link has been added, which provides more flexibility and enables more designs.


## Testing instructions
1. Existing `Post Excerpt` blocks should work as before(both editor and front end)
2. Existing Post Excerpt blocks which hadn't any `read more` text set should migrate and hide the editable field in the editor. In frontend we didn't show anything anyway if it wasn't set.
3. Add new `Post Excerpt` blocks and observe that the field is not visible at the insertion and is displayed if we toggle the attribute at the Inspector controls.
